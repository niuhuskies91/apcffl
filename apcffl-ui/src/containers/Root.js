import React from 'react';
import { hot } from 'react-hot-loader/root'

import App from 'Components/App';
import 'Styles/Base.scss';

export class Root extends React.PureComponent {
  render () {
    return (
      <App />
    );
  }
}

export default hot(Root);
