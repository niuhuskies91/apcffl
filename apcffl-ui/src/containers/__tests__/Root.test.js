import React from 'react';

import { Root } from '../Root';

describe('Root', () => {
  it('renders', () => {
    const { asFragment } = render(<Root />);
    expect(asFragment()).toMatchSnapshot();
  });
});
