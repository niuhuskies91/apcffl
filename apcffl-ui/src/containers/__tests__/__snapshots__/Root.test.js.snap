// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root renders 1`] = `
<DocumentFragment>
  <section>
    <h1>
      Frigg off, Barb!
    </h1>
  </section>
</DocumentFragment>
`;
